### ISSUE TITLE
params error!

### ISSUE THREAD

**Issue by v-florinskiy** [tiagosiebler/bybit-api#140]
i try to place order, but get error:

`{ ret_code: 10001,
  ret_msg: 'params error!',
  ext_code: '',
  ext_info: '',
  result: null,
  time_now: '1647448210.677814' }`

what did i do wrong?
my code

```javascript
const { InverseFuturesClient } = require('bybit-api');
const API_KEY = '---';
const PRIVATE_KEY = '---';
const useLivenet = false;
const client = new InverseFuturesClient(
  API_KEY,
  PRIVATE_KEY,
  useLivenet,
);
run();
async function run(){
     const marketQty = parseInt(1);
     const params = {
      side: 'Sell',
      symbol: "SOLUSDT",
      order_type: 'Market',
      qty: marketQty,
      time_in_force: 'GoodTillCancel',
    }
    console.log(await client.placeActiveOrder(params));
}
```

**Comment by tiagosiebler** [comment]
Could it be the invalid symbol? These are the inverse futures symbols:

<img width="608" alt="Screenshot 2022-03-16 at 16 55 03" src="https://user-images.githubusercontent.com/14090872/158644979-0f68dd09-08a2-45db-aae5-4f68e884098b.png">

**Comment by v-florinskiy** [comment]
i'm sorry, i used wrong futures type, Instead linear.
But placeActiveOrder in LinearClient require default parameters "reduce_only" and "close_on_trigger".
In your code they are not require. I think this need correct.

`reduce_only?: boolean;
    close_on_trigger?: boolean;`

**Comment by tiagosiebler** [comment]
> i'm sorry, i used wrong futures type, Instead linear. But placeActiveOrder in LinearClient require default parameters "reduce_only" and "close_on_trigger". In your code they are not require. I think this need correct.
> 
> `reduce_only?: boolean; close_on_trigger?: boolean;`

Yes, Someone actually contributed a fix #136 that's taken me too long to review. Just merged it, it'll be on npm in the latest release, shortly.
